en/docs/get-started/develop-event-integration.md

```diff
@@ -31,26 +31,22 @@ Before you begin, make sure you have the following:
     <a href="{{base_path}}/assets/img/get-started/develop-event-integration/add-configurables.gif"><img src="{{base_path}}/assets/img/get-started/develop-event-integration/add-configurables.gif" alt="Add Configurations" width="80%"></a>
 
 5. Go to the **Design View** by clicking the Home icon on the top left corner and click **Add Artifact**.
-6. Select **RabbitMQ Event Handler**. Choosing the **Event Integration** from the Devant console disables the other options.
-7. Provide the name of the **RabbitMQ Configuration** as `eventListener`.
-8. Select previously defined `host` and `port` configuration variables for the **Host** and **Port**.
-9. Then, expand the **Advanced Configurations** and enter the following configurables. Then click **Next**.
-
-    | Field                   | Value        |
-    |-------------------------|--------------|
-    | **username**            | `username`   |
-    | **password**            | `password`   |
-
-10. Add `Orders` as the **Queue Name** and click **Create**. If there is no queue named `Orders` in RabbitMQ server, this will create a new queue with this name. 
-
+6. Select **RabbitMQ Event Integration**. Choosing the **Event Integration** from the Devant console disables the other options.
+7. Add `Orders` as the **Queue Name** and click **Create**. If there is no queue named `Orders` in RabbitMQ server, this will create a new queue with this name.
```

Comment:
```suggestion
7. Add `Orders` as the **Queue Name** and click **Create**. If there is no queue named `Orders` in the RabbitMQ server, this will create a new queue with this name.
```